src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -167,6 +171,44 @@ def custom_serialize_conditions(conditions_dict):
     return featurefilterdict
 
 
+def map_featureflag_to_keyvalue(featureflag):
+    '''
+        Helper Function to convert FeatureFlag object to KeyValue object
+
+        Args:
+            featureflag - FeatureFlag object to be converted
+
+        Return:
+            KeyValue object
+    '''
+    try:
+        enabled = False
+        if featureflag.state in ("on", "conditional"):
+            enabled = True
+
+        feature_flag_value = FeatureFlagValue(id_=featureflag.key,
+                                              description=featureflag.description,
+                                              enabled=enabled,
+                                              conditions=featureflag.conditions)
+
+        set_kv = KeyValue(key=FEATURE_FLAG_PREFIX + featureflag.key,
+                          label=featureflag.label,
+                          value=json.dumps(feature_flag_value,
+                                           default=lambda o: o.__dict__,
+                                           ensure_ascii=False),
+                          content_type=FEATURE_FLAG_CONTENT_TYPE,
+                          tags={})
+
+        set_kv.locked = featureflag.locked
+        set_kv.last_modified = featureflag.last_modified
+
+    except:
+        logger.debug("Exception while converting feature flag to key value:\n%s\n", featureflag)
```

Comment:
One question here is if the exception raised will throw clear error message.
If not, what about make logger.debug message here be part of exception message?

Reply:
Done